use-cases/azure.md

```diff
@@ -0,0 +1,240 @@
+[
+  {rabbit, [
+   {auth_backends, [rabbit_auth_backend_oauth2, rabbit_auth_backend_internal]}
+  ]},
+  {rabbitmq_management, [
+     {oauth_enable, true},
+     {oauth_client_id, "PUT YOUR AZURE AD APPLICATION ID"},
+     {oauth_client_secret, "PUT YOUR AZURE AD APPLICATION SECRET"},
+     {oauth_provider_url, "https://login.microsoftonline.com/AZURE_AD_TENANT_ID"}
+ ]},
+ {rabbitmq_auth_backend_oauth2, [
+   {resource_server_id, <<"PUT YOUR AZURE AD APPLICATION ID">>},
+   {extra_scopes_source, <<"roles">>},
+   {key_config, [
+     {jwks_url, <<"PUT YOUR AZURE AD JWKS URI VALUE">>}
+   ]}
+ ]}
+].
+```
+
+> :warning: Please update the file available in this tutorial ([here](../conf/azure/rabbitmq.config)), as it will be automatically loaded in the RabbitMQ instance that we are going to deploy later in this tutorial
+
+### Generate SSL certificate and key
+> :warning: Remember when you have registered your app on Azure AD that it only allows **https** protocol for OAuth2 **Redirect URI**? We will thus need to enable HTTPS for RabbitMQ Management UI amd its underlying API.
+
+For the purpose of this tutorial, we can generate a self-signed certificate/key pair.
+
+Run the following command (depending on your config, you may have to be root):
+```
+make build-azure
+```
+
+This generates the following files in `conf/azure`:
+- **rabbitmq-ca.**crt**: a custom certificate authority that is used to generate and sign a self signed certificate for RabbitMQ
+- **rabbitmq.crt**: a self-signed certificate (cn=localhost)
+- **rabbitmq.key**: the private key associated to the `rabbitmq.crt` certificate
+
+:arrow_right: These files will be mounted into the `rabbitmq` docker container in the next steps of this tutorial, where they will be used to configure HTTPS for the RabbitMQ Management UI/API
+
+## Start RabbitMQ
+
+Run the following commands to run RabbitMQ docker image with the latest changes from `oidc-integration` branch with commit tag `69a4159f3482e5212d364f499b2ca2e05bede0ca`.
+
+> :bulb: All the commits associated to `oidc_integration` branch are available [here](https://github.com/rabbitmq/rabbitmq-server/commits/oidc-integration). Don't hesitate to have a look a it to get the very last commit, as the one advertised in this tutorial may not stay the last one forever!   
+
+```
+export IMAGE_TAG=69a4159f3482e5212d364f499b2ca2e05bede0ca-otp-min
+export IMAGE=pivotalrabbitmq/rabbitmq
+export MODE=azure
+make start-rabbitmq 
+```
+:arrow_right: This starts a docker container named `rabbitmq`, with RabbitMQ Management UI/API with HTTPS enabled, and configured to use your Azure AD as OAuth2 Authentication Backend, based on the information you provided in `rabbitmq.config` in the previsous steps of this tutorial.
+
+## Verify RabbitMQ Management UI access
+
+Go to RabbitMQ Management UI `https://localhost:15671`. Depending on your browser, ignore the security warnings (raised by the fact that we are using a self-signed certificate) to proceed. 
```

Comment:
very minor: I would not wrap the uri `https://localhost:15671` in single quotes so that the reader can simply click and redirect to the management ui.